runtime/gc_vlhgc/IncrementalGenerationalGC.cpp

```diff
@@ -1976,6 +1978,9 @@ MM_IncrementalGenerationalGC::postConcurrentUpdateStatsAndReport(MM_EnvironmentB
 
 	stats->_bytesScanned = bytesConcurrentlyScanned;
 	stats->_terminationWasRequested = _forceConcurrentTermination;
+	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
+	env->_cycleState = &_persistentGlobalMarkPhaseState;
+	static_cast<MM_CycleStateVLHGC*>(env->_cycleState)->_vlhgcIncrementStats._markStats._endTime = omrtime_hires_clock();
```

Comment:
Do we need to set the end time? Seems like TS is passed to the trigger, we can make use of that.